apps/omar-geoscript-app/grails-app/services/omar/geoscript/HeatMapService.groovy

```diff
@@ -90,10 +103,53 @@ class HeatMapService {
             {
                 if((isValidJson(result.hits.hits.getAt(i)._source.message))) {
                     Feature feature = writer.newFeature
-                    Map<String, Object> map = new ObjectMapper().readValue(result.hits.hits.getAt(i)._source.message, HashMap.class);
+                    Map<String, Object> logmap = new ObjectMapper().readValue(result.hits.hits.getAt(i)._source.message, HashMap.class);
+
+                    def minx = logmap.get("bbox").minX
```

Comment:
I think you should be able to just call the objects directly from logmap by doing something like "def minx = logmap.bbox.minX" because Groovy automatically converts that to a .get method call. Go ahead and try that out and see how it works. Same should be able to be done below with logmap.get("params")

Reply:
Done